usage/app/lib/Config.scala

```diff
@@ -80,7 +80,7 @@ object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
   )
 
   private val iamClient: AmazonIdentityManagement =
-    new AmazonIdentityManagementClient(credentialsProvider)
+    new AmazonIdentityManagementClient(awsCredentials)
```

Comment:
That creates a security token is invalid exception. I assumed it was a problem with using credentials from janus to perform the request.